### PR TITLE
finally done

### PR DESCRIPTION
**Pull request by HosenNaser**
I know  am too late but, I was so busy at work in last two days, and finally completed the task

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
+//-----------------------Tester----------------------------
+function assertTrue(testFunction, functionParams) {
```
Comment (EliasSalom) [inline js/script.js:2]: why the function is taking 2 parameters?

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  } catch (error) {
+    console.log("the test testEmailValidationFail has FAILED!");
+    console.log(">>> " + error);
+  }
+}
+
+runTest();
+
+//-----------------------Check Email Function----------------------------
+function checkEmail(email) {
```
Comment (EliasSalom) [inline js/script.js:101]: all that I need for my task is just this part please move all these functions to another file !!!

**Review by EliasSalom — APPROVED** [review]
please just update my review on your code

File: js/script.js (hunk 2 above)
Comment (HosenNaser) [reply js/script.js:2]: the tester has been removed

**Review by EliasSalom — COMMENTED** [review]
good work, please continue in this way!

**Pull request merged**